torch/fx/experimental/symbolic_shapes.py

```diff
@@ -100,8 +100,15 @@ def sym_int(a):
         return a.__sym_int__()
     elif isinstance(a, torch._C.SymIntNode):
         return a
+    elif isinstance(a, torch._C.SymFloatNode):
+        return math.floor(a)
```

Comment:
This is making the lint very unhappy with 
```
    Argument 1 to "floor" has incompatible type "SymFloatNode"; expected
    "SupportsFloat"
```
But I'm not sure how to fix it... cc @ezyang who looked into making these look like int/float

Reply:
I think the type annotation for math.floor is wrong; the mypy annotation says it should SupportsFloat, but this is not required; in fact only `__floor__` method is needed. We DO NOT want to add `__float__` to sym float. So I recommend suppressing this lint.

Reply:
Ok!